stl/inc/xiosbase

```diff
@@ -547,9 +547,9 @@ private:
     fmtflags _Fmtfl; // format flags
     streamsize _Prec; // field precision
     streamsize _Wide; // field width
-    _Iosarray* _Arr; // pointer to first node of long/pointer array
-    _Fnarray* _Calls; // pointer to first node of call list
-    locale* _Ploc; // pointer to locale
+    _Iosarray* _Arr{nullptr}; // pointer to first node of long/pointer array
+    _Fnarray* _Calls{nullptr}; // pointer to first node of call list
+    locale* _Ploc{nullptr}; // pointer to locale

```

Comment:
Note: this changes the set of "valid" states, it should be fine since tests pass and you check for deletion

Reply:
I also think it's fine because zero/null is a strict subset of garbage (i.e. zero-initialization is never worse for correctness than default-initialization).